InnerEye/Common/generic_parsing.py

```diff
@@ -122,7 +122,12 @@ def _get_basic_type(_p: param.Parameter) -> Union[type, Callable]:
             :return: Type
             """
             if isinstance(_p, param.Boolean):
-                p_type: Union[type, Callable] = lambda x: (str(x).lower() == 'true')
+                def parse_bool(x: str) -> bool:
+                    sx = str(x).lower()
+                    if sx == 'true' or sx == 'false':
```

Comment:
maybe add 0 and 1 as OK values too?

Reply:
I've added: y/n, yes/no, 1/0, t/f, on/off because I can imagine someone using them